index.js

```diff
@@ -17,12 +35,20 @@ exports.registerFor = function (filepath, cwd) {
     cwd = path.dirname(path.resolve(filepath));
```

Comment:
this block can be moved below the moduleName lookup, it won't be needed unless a module exists.